ngraph/src/ngraph/frontend/onnx_import/core/graph.cpp

```diff
@@ -190,8 +193,29 @@ namespace ngraph
         {
             const auto ng_node_factory =
                 m_model->get_operator(onnx_node.op_type(), onnx_node.domain());
-
-            const auto ng_node_vector = ng_node_factory(onnx_node);
+            NodeVector ng_node_vector;
+            try
+            {
+                ng_node_vector = ng_node_factory(onnx_node);
+            }
+            catch (const ::ngraph::onnx_import::error::OnnxNodeValidationFailure& exc)
+            {
+                // Do nothing OnnxNodeValidationFailure exception already has ONNX node information.
+                throw;
+            }
+            catch (const std::exception& exc)
+            {
+                std::string msg_prefix = error::detail::get_error_msg_prefix(onnx_node);
+                throw ngraph_error(msg_prefix + ":\n" + std::string(exc.what()));
+            }
+            catch (...)
+            {
+                std::string msg_prefix = error::detail::get_error_msg_prefix(onnx_node);
+                // Since we do not now anything about current exception data type we can only
+                // notify user this way.
```

Comment:
```suggestion
                // Since we do not know anything about current exception data type, we can only
                // notify user in this way.
```